dev/build_release.py

```diff
@@ -594,3 +595,14 @@ def do_build(cls, options, build_number, container_builder):
 
       print '\nFINISHED writing release to {rep}'.format(
         rep=options.bintray_repo)
+
+  @classmethod
+  def main(cls):
+    parser = argparse.ArgumentParser()
+    cls.init_argument_parser(parser)
+    options = parser.parse_args()
+    # builds debians only
+    cls.do_build(options, '', None)
```

Comment:
Seems like build_number and container_builder should be keyword arguments that default to None so you only specify them when you want to override the defaults.